### DISCUSSION TITLE
Installation of widevine on windows does not match installation of ungoogled chromium.

### DISCUSSION THREAD

**Discussion by miguelcarcamov** [Eloston/ungoogled-chromium#1648] [Q&A]
When going to the FAQ page the website to install widevine says that we need to download a .zip file and unzip the file and put the files on chromium folders. However, these folders don't exist for ungoogled-chromium (Windows). Can anyone provide instructions on how to do this?

Cheers

**Comment by networkException** [comment]
I just tested this on a fresh Windows VM and it worked for me. I did the following:

- Download the zip from <https://dl.google.com/widevine-cdm/4.10.2209.1-win-x64.zip>
- Go to `\Users\<username>\AppData\Local\Chromium\<version>\`
- Create a folder called `WidevineCdm`
- Copy `LICENCE` and `manifest.json` from the zip in there
- Create a two directories in `WidevineCdm`: `_platform_specific` and in there `win_x64`

> So the structure would look like
> ```
> WidevineCdm
> ├── LICENCE
> ├── manifest.json
> └── _platform_specific
>     └── win_x64
>         ├── widevinecdm.dll
>         ├── widevinecdm.dll.lib
>         └── widevinecdm.dll.sig

- Copy the remaining files from the zip in there
- Restart chromium
- Test on <https://demo.castlabs.com/> (click on the video with DRM in the tag)
